### ISSUE TITLE
How to overclock more then 1.9GHz?

### ISSUE THREAD

**Issue by leonpano2006** [borongyuan/XavierNXOverclock#2]
This for me is not real overclock
Real overclock need to be gpu more then 1.1GHz
CPU need to be 2.xGHz

**Comment by borongyuan** [comment]
Yes, this is just an easy way to break out of nvpmodel's presets. To unlock higher frequencies, you may need to patch and rebuild the kernel. Given the cooling capabilities of most products, it doesn't seem like much sense to do so.
